guide/packages.md

```diff
@@ -154,6 +182,27 @@ The Membrane docker ([Docker Hub](https://hub.docker.com/r/membrane/membrane), [
+| ------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `membrane-webrtc-js` | JS/TS client library for Membrane RTC Engine | [![NPM version](https://img.shields.io/npm/v/@membraneframework/membrane-webrtc-js)](https://www.npmjs.com/package/@membraneframework/membrane-webrtc-js) [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane-webrtc-js/)                                                                                                                                                                                                                           |
+| `membrane-webrtc-ios` | Membrane WebRTC client compatible with Membrane RTC Engine | [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane-webrtc-ios)                                                                                                                                                                                                                           |
+| `membrane-webrtc-android` | Membrane WebRTC client compatible with Membrane RTC Engine | [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane-webrtc-android)                                                                                                                                                                                                                           |
+| `react-native-membrane-webrtc` | React Native wrapper for `membrane-webrtc-ios` and `membrane-webrtc-android` | [![NPM version](https://img.shields.io/npm/v/@membraneframework/react-native-membrane-webrtc)](https://www.npmjs.com/package/@membraneframework/react-native-membrane-webrtc)  [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/react-native-membrane-webrtc)                                                                                                                                                                                                                           |
```

Comment:
Those packages, alongside with membrane_rtc_engine_timescaledb and membrane_rtc_engine itself, are going to be moved to jellyfish so I am not sure wheter we should mention them here? Maybe we should mention about jellyfish in general? Any thoughts?

Reply:
We could make a new page with Jellyfish packages or rename that section to Jellyfish (instead of WebRTC) and explain what Jellyfish is.

Reply:
I would opt for the solution proposed by Dominik ;)